terraform/helm/forge/values.yaml

```diff
@@ -16,11 +16,14 @@ forge:
   tolerations: []
   affinity: {}
 
+continuous_test:
+  enabled: false
+  intervalMins: 15
```

Comment:
instead of the intervalMins I would've just put the raw schedule value here imo. but you can keep it.

Reply:
Our other stuff is scheduled in this way for simplicity, so I just kept that format. Raw schedule would be more flexible, but i'll keep this for now...